tempesta_fw/ss_skb.c

```diff
@@ -788,20 +804,15 @@ __skb_fragment(struct sk_buff *skb_head, struct sk_buff *skb, char *pspt,
 		d_size = skb_frag_size(frag);
```

Comment:
We do not update `fragn` for the case `len > 0 && !offset` above. Is this correct? Need a review from @i-rinat since he fixed many bugs recently with iterator frags.